src/components/Autocomplete/Autocomplete.test.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Autocomplete from './Autocomplete';
+
+describe('Autocomplete', () => {
+  it('should render autocomplete component', () => {
+    const data = [];
+    render(<Autocomplete data={data} id={'1'} />);
+    expect(screen.getByLabelText(/Search input/)).toBeInTheDocument();
+  });
+});
```

Comment:
Add test cases to check functionality, e.g. does it suggest sentences which are given in `data`:
```
  it('should suggest sentences.', () => {
    const data = [
      {
        title: 'Russia has no plans to occupy Ukraine.',
        year: 2022,
      },
    ];
    render(<Autocomplete data={data} id={'1'} />);
    userEvent.click(screen.getByLabelText(/Search input/));
    expect(
      screen.getByText('Russia has no plans to occupy Ukraine.')
    ).toBeInTheDocument();
  });
```

Reply:
Thank you for the comment, Done